Description: Define _GNU_SOURCE for strptime_l
 Without this we get -Wimplicit-function-declaration warnings from GCC on
 non-Linux glibc-based architectures.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: https://github.com/kazu-yamamoto/unix-time/pull/11
Last-Update: 2013-06-15

Index: b/cbits/conv.c
===================================================================
--- a/cbits/conv.c
+++ b/cbits/conv.c
@@ -7,6 +7,7 @@
 #define _BSD_SOURCE
 #elif HAVE_STRPTIME_L
 #define THREAD_SAFE 1
+#define _GNU_SOURCE
 #else
 #define THREAD_SAFE 0
 #endif
